test/location.cc

```diff
@@ -101,102 +101,112 @@ void test_from_json() {
   uint64_t way_id = 12987234107;
   std::string date_time = "2015-05-06T08:00";
 
-  try {
-    Location loc = Location::FromJson("something");
-    throw std::runtime_error("This should have been malformed json");
-  } catch (...) {
+  std::vector<Location::ParseMethod> methods{Location::ParseMethod::PTREE,
+    Location::ParseMethod::RAPIDJSON};
+
+  for(const auto& m : methods) {
+    try {
+      Location loc = Location::FromJson("something", m);
+      throw std::runtime_error("This should have been malformed json");
+    } catch (...) {
+    }
```

Comment:
@kevinkreiser  Why this do something that will just be trashed whatever, then throw, and all that is catched nothing? That's just useless code, no?